code/mission/missioncampaign.cpp

```diff
@@ -1822,7 +1822,12 @@ void mission_campaign_exit_loop()
 
 	// set things up for next mission
 	mission_campaign_next_mission();
```

Comment:
Is this call still correctly placed?

Reply:
It should be.  In any case, there's no harm in calling it even if the campaign has ended.